src/test/regress/expected/citus_split_shard_by_split_points_deferred_drop.out

```diff
@@ -129,4 +118,7 @@ SELECT relname FROM pg_class where relname LIKE '%table_to_split_%' AND relkind
 -- Test Cleanup
 \c - postgres - :master_port
 DROP SCHEMA "citus_split_shard_by_split_points_deferred_schema" CASCADE;
```

Comment:
can you add `SET client_min_messages TO WARNING;` before the DROP SCHEMA call to make these notices go away. That way there's less churn in the future when stuff gets added.

Reply:
sure, added